### ISSUE TITLE
Doesn't work from explorer if there are multiple dots in filename

### ISSUE THREAD

**Issue by kurt-rutabaga** [altenull/json-sea-vscode#1]
If I have a file on disk, say thing.testdata.json, the right click doesn't register.

**Comment by altenull** [comment]
@kurt-rutabaga 
Sorry for being late. I just checked but it seems that it has no problem on opening multiple dots in filename.

Here is testing result. (`thing.testdata.json` file)
<img width="600" alt="multiple dots filename test" src="https://github.com/altenull/json-sea-vscode/assets/26947777/ee05dc01-b83a-4f5a-9d07-62e2b46d597d">

Can't you open it with right click? Then how is command palette?

**Comment by altenull** [comment]
@kurt-rutabaga 

Thanks to your repoting I found a bug in other place.
When object property name includes dots in it, the wrong node path is showing.

<img width="600" alt="dots in object property name" src="https://github.com/altenull/json-sea-vscode/assets/26947777/7ff80f37-ec6e-473f-a9d0-089d4c05023b">

The issue you reported seems okay, so I'm going to resolve a bug that I found, the dots in object property name.
